testsuite/tests/kuadrant/gateway/test_dns.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-pytestmark = [pytest.mark.kuadrant_only]
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.dns, pytest.mark.tls]
```

Comment:
I'd call marks dnspolicy and tlspolicy as tls could also refer to our authorino tests for certificates

Reply:
good point